packages/vkui/src/components/FormItem/FormItem.tsx

```diff
@@ -77,7 +82,14 @@ export const FormItem = ({
   const wrappedChildren = (
     <React.Fragment>
       {hasReactNode(top) && (
-        <Subhead className={styles['FormItem__top']} Component={topComponent} htmlFor={htmlFor}>
+        <Subhead
+          className={classNames(
+            styles['FormItem__top'],
+            topMultiline && styles['FormItem__top-multiline'],
```

Comment:
nit:

```suggestion
            topMultiline && styles['FormItem__top--multiline'],
```

Так как это модификатор для блока `FormItem__top` в терминах [BEM](https://en.bem.info/methodology/block-modification/#adding-multiple-modifiers)

То же и в CSS файле.

Reply:
Done